packages/back-end/package.json

```diff
@@ -28,7 +28,7 @@
     "@databricks/sql": "^1.1.0",
     "@google-cloud/bigquery": "5",
     "@google-cloud/storage": "^5.20.5",
-    "@growthbook/growthbook": "^0.26.0",
+    "@growthbook/growthbook": "^0.27.0",
```

Comment:
I can't remember if we need to hold off on this change until the NPM packages are live, or whether it pulls from the same monorepo. Might be good to check before launching the change.

Reply:
I checked with Jeremy and he mentioned that the yarn workspaces will resolve the local package and do the yarn linking behind the scenes. The recent commit 9591d3a994773d7971c4e31e3ebe9b3e290f6899 removed the NPM registry item from the yarn.lock file, so I think that's an indication that it was working properly.

Reply:
This used to be the case where there was a very strict order of operations required when making SDK changes.  I fixed this recently to be a lot more forgiving.  So we can merge to main and release to npm in any order and it just works.